### PR TITLE
Improve tools for building and GovCMS 1.1.1

### PR DESCRIPTION
**Pull request by simesy**
This is a satis update for govcms 1.1.1. I have split out the docs to a separate maintenance doc, and there are two new scripts to help resolve issues with how we use Satis.

### CODE REVIEW

**Comment by simesy** [comment]
Output of new debug scripts.

Screen Shot 2020-03-31 at 23 52 47
Screen Shot 2020-03-31 at 12 47 31

File: scripts/check-missing.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+cp composer.json composer-copy.json && cat composer-copy.json \
+  | jq .repositories='{"drupal":{"type":"composer","url": "https://packages.drupal.org/8"},"custom":{"type":"path","url":"custom/composer"},"govcms":{"type":"composer","url":"http://localhost:4142"}}' \
+  | tee composer.json > /dev/null
+COMPOSER_MEMORY_LIMIT=-1 composer update
+
+# Get a list of desired versions from the world.
+target_list=$(composer info --format=json | jq -r '.installed[] | "\(.name) \(.version)"' | sort)
+
+# Point to the local satis build.
+composer config secure-http false
```
Comment (tobybellwood) [inline scripts/check-missing.sh]: I've had to move this before the composer update in L17, otherwise it won't work?

Comment (simesy) [reply scripts/check-missing.sh]: I agree, I don't know how it was working like that

**Comment by tobybellwood** [comment]
Can ahoy verify be run per-branch? It's be good to check each branch without being blocked on another one?

**Comment by simesy** [comment]
I've separated master/develop/stable into three sets of commands:

- ahoy build-{branch}
- ahoy debug-{branch}
- ahoy verify-{branch}

Screen Shot 2020-04-09 at 15 30 04

**Comment by simesy** [comment]
I picked up a couple of issues in building master/develop too. They need `composer config prefer-stable false`. And if `composer show` fails it was failing the check-missing script.

**Review by tobybellwood — APPROVED** [review]
LGTM - sensible set of commands, well documented

**Pull request merged**